Library/Homebrew/tap.rb

```diff
@@ -302,6 +303,11 @@ def formula_dir
     @formula_dir ||= [path/"Formula", path/"HomebrewFormula", path].detect(&:directory?)
   end
 
+  # path to the directory of all casks for caskroom/cask {Tap}.
+  def cask_dir
+    @cask_dir ||= [path/"Casks", path].detect(&:directory?)
```

Comment:
This should just be `path/Casks` as I don't remember cask supporting put cask file in the top directory, right?